ui/selectors/assets.ts

```diff
@@ -138,6 +139,14 @@ export const getTokenBalancesEvm = createDeepEqualSelector(
             balance !== '0' ||
             (token.isNative && isOnCurrentNetwork)
           ) {
+            // title is used for sorting. We override native ETH to Ethereum
```

Comment:
TY for comment.
Gosh all these small edge cases feel really prickly - I don't know of a nice way immediately to improve this.

Reply:
A few steps I've considered:

1. Ask Token API team to handle this override at the API layer (shouldn't it actually be `Ether` anyway?)
2. Eventually migrate the evm token data to the multichain state. This sounds hard, but would make things way more consistent

Reply:
Nice ideas, I really vibe with the 1st point (override or additional properties at the API layer).
The second point is a bit of a unknown to me, I might catchup with you to explain this to me 😄